lib/mayaUsd/ufe/Global.cpp

```diff
@@ -199,11 +204,13 @@ MStatus initialize()
         = displayInfoMessage;
     dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::kWarning)]
         = displayWarningMessage;
-    dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::KError)]
+    dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::kError)]
         = displayErrorMessage;
     dccFunctions.startWaitCursorFn = mayaStartWaitCursor;
     dccFunctions.stopWaitCursorFn = mayaStopWaitCursor;
     dccFunctions.defaultMaterialScopeNameFn = defaultMaterialsScopeName;
+    dccFunctions.extractTRSFn = MayaUsd::ufe::extractTRS;
+    dccFunctions.transform3dMatrixOpNameFn = getTransform3dMatrixOpName;
```

Comment:
Two new (optional) DCC functions that we set for MayaUsd. UsdUfe provides a default for both.